### ISSUE TITLE
Proper tree/subtree printing

### ISSUE THREAD

**Issue by Oleksii-Kshenskyi** [Oleksii-Kshenskyi/leviathanK#5]
At this point, some kind of printing of the nodes has been already implemented in some form.

This task is for making leviathanK print out subtrees in a comprehensive and pretty form. The approximate output format is as follows:
```
<node_path> : <node_value>
<node_path> : <node_value>
```
Example:
```
path : 1
path/to : 2
path/to/the : 3
path/to/the/node : 4
other : 5
other/path : 6
```
This function has to take a tree/subtree as an argument and print out the entirety of it.

**NOTE:** As the direct result of implementing #3, printing trees already works as intended. The only thing that's left is to make printing empty nodes and pointers to tree nodes optional (via additional parameter). Introducing a parameter for path_tree_print() to choose whether to print empty nodes and node pointers is going to be done in the scope of this issue.

**TL;DR:** implement an additional choice in path tree printing: whether to print empty nodes and pointers to all the nodes (basically debugging information) or not.

**Comment by Oleksii-Kshenskyi** [comment]
[WEIGHT: 4.924 U @ 02:03:06]
